### PR TITLE
Update README to include more information on running tests

### PR DESCRIPTION
**Pull request by calebmshafer**
Took an initial step to documenting how to run the tests. No one outside of the team will be able to run them yet since we don't show/explain how to create each test iModel, but that can come later.

Consolidated the setup for test iModels and Contexts to be similar to how I updated the backend tests. Cleaned up the unused portions as well. We probably should have some sort of internal only shared package that maintains all of these lists and tools...

### CODE REVIEW

File: full-stack-tests/core/README.md [hunk 1]
```diff
@@ -1,7 +1,34 @@
...
+1. Populate
+
+    ```sh
+    # User registered with iTwin Platform
+    imjs_test_regular_user_name=
+    imjs_test_regular_user_password=
+
+    # Register a SPA client from https://developer.bentley.com/my-apps.
+    imjs_oidc_browser_test_client_id=
+    imjs_oidc_browser_test_redirect_uri=
```
Comment (pmconne) [inline full-stack-tests/core/README.md:30]: ur=what?

Comment (calebmshafer) [reply full-stack-tests/core/README.md:30]: It depends on the client registration. I'll make a note of that.

Comment (pmconne) [reply full-stack-tests/core/README.md:30]: ?

Comment (calebmshafer) [reply full-stack-tests/core/README.md]: Updated to make it more clear. I set it to a default redirect_uri, although you can technically change that.

**Review by pmconne — APPROVED** [review]

**Pull request merged**